src/components/ebay-carousel/README.md

```diff
@@ -14,25 +14,17 @@ This component will bundle different resources depending on Lasso flags provided
 Use `touch` for touch devices, and `no-touch` for non-touch devices. Without flags, `no-touch` is assumed.
 [//]: # (TODO: `touch` should ideally be default, but currently the js behavior doesn't execute with native scrolling)
 
-Note: The `touch` carousel with type="continuous" will use native scrolling. This behavior is used instead of our controlled movement, so some functionality is limited. In this case, the carousel will not respond to `index` changes, and will not emit events.
-
 ## ebay-carousel Attributes
 Name | Type | Stateful | Description
 --- | --- | --- | ---
-`type` | String | No | "continuous" (default) or "discrete"
```

Comment:
How will this work on mobile devise if there is only 1 item per slide and I want that to be continuous? Meaning I am using natural mobile horizontal scroll and can leave the item in an in-between state.

Reply:
So if you want 1 item per slide you can use that option. In this PR on mobile discrete carousels will also work in native scroll mode in mobile. After the scroll has completed it will then animate to the closest slide.

Items never stay in an in between state. If that is a requirement then we can change that fairly easily but I think it works well currently.